lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +81,40 @@ def compare_profiles(
     :param profile_to_compare: a dictionary of a profile to compare the unknown profile to
     :return: the distance between the profiles
     """
+    if isinstance(unknown_profile, dict) == False or isinstance(profile_to_compare, dict) == False:
+        return None
+    if 'name' not in unknown_profile or  'freq' not in unknown_profile:
+        return None
+    if 'name' not in profile_to_compare or 'freq' not in profile_to_compare:
+        return None
+
+    sorted_profile_to_compare = dict(sorted(profile_to_compare.items()))
+    sorted_unknown_profile = dict(sorted(unknown_profile.items()))
```

Comment:
Dictionaries do not guarantee a specific order. Please find another way to ensure the same order of the frequencies, as this is an insecure way to do it.

Reply:
fixed